### ISSUE TITLE
What is "Open Seats"?

### ISSUE THREAD

**Issue by bjornmeansbear** [GD399-OSD/openseats#1]
@mazzybell and @colemcdonnell 

Mazzy created this repo last week to house whatever investigations you two do on the open furniture front — let's figure out how to make things happen here!

**Comment by mazzybell** [comment]
I have some things to post and I tried to make a new file but github won't let me ? Help ! 

Edit: actually it worked but I feel like I formatted it wrong.

**Comment by bjornmeansbear** [comment]
@mazzybell is this the file: https://github.com/GD399-OSD/openseats/blob/master/links

We might get the best results out of getting you using the github desktop app so that you can work on your computer and sync things here when convenient?
https://desktop.github.com/

**Comment by mazzybell** [comment]
Yeah that’s it. I just don’t understand why it looks different than
everything else. I guess you can just explain it to me tomorrow. Oy.

**Comment by bjornmeansbear** [comment]
What looks "different" just that it is plaintext links?

**Comment by bjornmeansbear** [comment]
I fixed it — I renamed the file to have an extension, .md (markdown) — computers don't like files to not have extensions as it doesn't tell them what to do w/ the text otherwise.

I also added hyphens to the beginning of the lines so as to make them show readily as a list
